README.md

```diff
@@ -1,38 +1,25 @@
 # Twitter-API-Account-Manager-Python3
-## Python 3 script for managing your twitter account using twitter official api's
 
-### Requirements
```

Comment:
Don't you think that it would be nice if we keep this title here? Either as a Requirements or as a Prerequisite?

Reply:
To be honest no, because I think users should install this project with `pip` not by install its dependencies manually.

Reply:
Hi guys
"QuickStart" is also good for title in here and it doesn't matter really :)